src/client/api/extension.d.ts

```diff
@@ -0,0 +1,256 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Later we can publish this as npm or github package, else users have to hand craft this.
For now lets get users to manually copy this over.